wallet/lib/pages/presenting_onboard_page/screens/accept_policy_screen.dart

```diff
@@ -0,0 +1,263 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:provider/provider.dart';
+import 'package:pylons_wallet/components/buttons/pylons_get_started_button.dart';
+import 'package:pylons_wallet/components/pylons_app_theme.dart';
+import 'package:pylons_wallet/generated/locale_keys.g.dart';
+import 'package:pylons_wallet/model/nft.dart';
+import 'package:pylons_wallet/pages/detailed_asset_view/widgets/nft_3d_asset.dart';
+import 'package:pylons_wallet/pages/presenting_onboard_page/viewmodel/accept_policy_viewmodel.dart';
+import 'package:pylons_wallet/pylons_app.dart';
+import 'package:pylons_wallet/utils/clipper_utils.dart';
+import 'package:pylons_wallet/utils/constants.dart';
+import 'package:pylons_wallet/utils/enums.dart';
+import 'package:pylons_wallet/utils/image_util.dart';
+import 'package:pylons_wallet/utils/route_util.dart';
+import 'package:pylons_wallet/utils/svg_util.dart';
+import 'package:shimmer_animation/shimmer_animation.dart';
+import 'package:url_launcher/url_launcher_string.dart';
+
+class AcceptPolicyScreen extends StatefulWidget {
+  final NFT nft;
+  final AcceptPolicyViewModel viewModel;
+
+  const AcceptPolicyScreen({Key? key, required this.nft, required this.viewModel}) : super(key: key);
+
+  @override
+  State<AcceptPolicyScreen> createState() => _AcceptPolicyScreenState();
+}
+
+class _AcceptPolicyScreenState extends State<AcceptPolicyScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(
+        children: [
+          SizedBox.expand(
+            child: getTypeWidget(
+              widget.nft,
+            ),
+          ),
+          Container(
+            height: double.infinity,
+            width: double.infinity,
+            color: Colors.grey.withOpacity(0.3),
+          ),
+          Padding(
+            padding: EdgeInsets.only(left: 8, right: 8, bottom: 8, top: MediaQuery.of(context).viewPadding.top),
+            child: SizedBox(
+              height: 100.h,
+              width: double.infinity,
+              child: ListTile(
+                leading: SvgPicture.asset(
```

Comment:
As per design and discussion the accept policy part feels like bottom sheet and user can not close it or go to next page without accepting it, so that button is there as per design but user can not click on them

Reply:
If the button is not working it should not be here. Discuss with designer on this,